libs/hmpb/src/render_ballot.tsx

```diff
@@ -496,9 +496,10 @@ export async function renderAllBallotsAndCreateElectionDefinition<
       }
       case 'cdf':
         return convertVxfElectionToCdfBallotDefinition(electionWithGridLayouts);
-      default:
-        /* istanbul ignore next */
+      default: {
+        /* istanbul ignore next - @preserve */
```

Comment:
Does this work here?

Reply:
Gotcha. Should we convert to this pattern in general (move the ignore comments into the default blocks)?

Reply:
It's probably the best option we have, though once we can use negative thresholds I'm not sure it'll matter much.